test/models/trip_test.dart

```diff
@@ -0,0 +1,369 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:motis_mitfahr_app/rides/models/ride.dart';
+import 'package:motis_mitfahr_app/util/search/position.dart';
+import 'package:motis_mitfahr_app/util/trip/trip.dart';
+
+import '../util/factories/drive_factory.dart';
+import '../util/factories/ride_factory.dart';
+import '../util/factories/trip_factory.dart';
+import '../util/mock_server.dart';
+import '../util/mock_server.mocks.dart';
+
+void main() {
+  final MockUrlProcessor tripProcessor = MockUrlProcessor();
+  //setup muss in jeder Testklasse einmal aufgerufen werden
```

Comment:
I would remove this comment everywhere, it's not even true

Reply:
missed that one, removed it in all the others